infra/ci/build.py

```diff
@@ -35,16 +35,21 @@
 LANGUAGES_WITH_COVERAGE_SUPPORT = ['c', 'c++', 'go']
 
 
-def get_changed_files():
+def get_changed_files_output():
   """Returns the output of a git command that discovers changed files."""
-  return subprocess.check_output(['git', 'diff', '--name-only',
-                                  'FETCH_HEAD']).decode()
+  main_branch = subprocess.check_output(
+      ['git', 'rev-parse', '--abbrev-ref', 'origin/HEAD']).strip().decode()
+  branch_commit_hash = subprocess.check_output(
+      ['git', 'merge-base', 'FETCH_HEAD', main_branch]).strip().decode()
```

Comment:
can we just specify origin/HEAD or do we actually need the rev-parse above?

Reply:
Actually no, so removing in https://github.com/google/oss-fuzz/pull/5056